x-pack/test/functional/page_objects/reporting_page.ts

```diff
@@ -140,8 +140,8 @@ export function ReportingPageProvider({ getService, getPageObjects }: FtrProvide
 
     async setTimepickerInDataRange() {
       log.debug('Reporting:setTimepickerInDataRange');
-      const fromTime = 'Sep 19, 2015 @ 06:31:44.000';
-      const toTime = 'Sep 19, 2015 @ 18:01:44.000';
+      const fromTime = 'Apr 27, 2019 @ 23:56:51.374';
+      const toTime = 'Aug 23, 2019 @ 16:18:51.821';
```

Comment:
These values have been wrong for a long time, but in this PR it matters because the content of the CSV is actually checked against a snapshot.